webpack/scenes/Tasks/TaskActions.js

```diff
@@ -95,12 +99,13 @@ export const pollTaskUntilDone = (taskId, extendedParams = {}, interval, orgId)
 
       if (isUnauthorized(action) || id !== orgId || loading) {
         reject(action.result);
-      } else if (action.response.pending) {
+      } else if (action.response && action.response.pending) {
+        // eslint-disable-next-line promise/prefer-await-to-then
```

Comment:
This is where I had some difficulty, and used `eslint-disable` for now.   @MariaAga I'm curious if your foreman tasks PR will affect this file?

Reply:
I dont think it will as I only added more info to the tasks API